rasa/nlu/test.py

```diff
@@ -398,6 +398,33 @@ def evaluate_response_selections(
     }
 
 
+def add_confused_intents_to_report(
+    report: Dict, target_intents: Iterable[Any], predicted_intents: Iterable[Any]
+) -> Dict:
+
+    from sklearn.metrics import confusion_matrix
+    from sklearn.utils.multiclass import unique_labels
+
+    cnf_matrix = confusion_matrix(target_intents, predicted_intents)
+
+    indices = np.argsort(cnf_matrix, axis=1)
```

Comment:
Can we add comments, e.g. "Sort by confidence scores, so we can later extract the top x wrongly classified intents."

Reply:
same to some of the things further down